src/powerbi-visual/pbiviz.json

```diff
@@ -4,7 +4,7 @@
     "displayName": "Speckle PowerBI Viewer",
     "guid": "specklePowerBiVisual",
     "visualClassName": "Visual",
-    "version": "2.0.0",
+    "version": "2.0.0.0",
```

Comment:
i dunno why we hadn't commited this in already, if it was so important for pbiviz